pages/embedded-portal.tsx

```diff
@@ -49,33 +61,32 @@ const EmbeddedPortal: NextPageWithLayout<Props> = ({
     setButtonText("Setting up Flatfile...");
   };
 
-  const flatfleSpace =
-    existingSpace?.flatfileData as unknown as FlatfileSpaceData;
+  const publishableKey = process.env.NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY;
+
+  if (!publishableKey) {
+    console.error("Missing NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY env var");
+    throw "Missing NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY env var";
+  }
+
   const [showSpace, setShowSpace] = useState(false);
+  const error = (error: string) => (
+    <div className="text-white text-lg font-semibold">{error}</div>
+  );
```

Comment:
i'd give this a background with an error object or something and give a standard error, "An error occurred opening the portal."

Reply:
+ console.error with the actual error

Reply:
edited

Reply:
![CleanShot 2023-06-14 at 19 06 38](https://github.com/FlatFilers/HCMShow/assets/12041898/addff244-8f1c-4ff2-afa4-09854ec60369)

Reply:
![CleanShot 2023-06-14 at 19 08 32](https://github.com/FlatFilers/HCMShow/assets/12041898/15540aeb-6e89-46a7-bee5-d04e89331992)